lib/admin_form_builder.rb

```diff
@@ -0,0 +1,20 @@
+class AdminFormBuilder < GenericFormBuilder
+  STANDARD_FIELDS.each do |method|
+    define_method(method.to_sym) do |field, *args|
+      options, *args = args
+      options ||= {}
+
+      options[:wrapper_html_options] = {class: "form-group"}.merge(options[:wrapper_html_options] || {})
+      options[:class] = Array(options[:class]) + ["form-control", "input-md-4"]
```

Comment:
It might be better to leave off the size class and specify it only when it's needed on a case by case basis. By default Bootstrap inputs stretch to fill up the entire width of their container.

I can't see a way to otherwise override input-md-4?